languages/ruby/exercises/concept/hashes/log_line.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module LogLevel
+  UNKNOWN = 0
+  INFO = 1
+  WARNING = 2
+  ERROR = 4
+end
+
+class LogLine
+  LEVEL = {
+    unknown: LogLevel::ERROR,
+    info: LogLevel::INFO,
+    warning: LogLevel::WARNING,
+    error: LogLevel::ERROR
+  }.freeze
+
+  def initialize(line)
+    @line = line
+  end
+
+  def level
+    LEVEL.fetch(level_string.downcase.to_sym, LogLevel::UNKNOWN)
```

Comment:
For consideration: is this the exercise that introduces `symbols` and is the student expected to know about this concept? If so, it should be mentioned in the `introduction.md` file. If not, the exercise should have a prerequisite on a concept that involves learning about `symbols`.

Reply:
Yes another good candidate for the introduction 👍